### PR TITLE
[improvement](regression test) add fe debug point

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/util/DebugPoint.groovy [hunk 1]
```diff
@@ -113,26 +122,25 @@ class DebugPoint {
...
 
     def getFEHostAndHTTPPort() {
-        assert false : 'not implemented yet'
+        return suite.getFrontendIpHttpPort()
     }
 
     def enableDebugPointForAllFEs(String name, Map<String, String> params = null) {
-        assert false : 'not implemented yet'
+        operateDebugPointForAllFEs({ host, port ->
+            println "enable debug point $name for FE $host:$port"
```
Comment (deardeng) [inline regression-test/framework/src/main/groovy/org/apache/doris/regression/util/DebugPoint.groovy:130]: log.info?

**Review by deardeng — APPROVED** [review]
LGTM

**Comment by yujun777** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/util/DebugPoint.groovy (hunk 1 above)
Comment (yujun777) [reply regression-test/framework/src/main/groovy/org/apache/doris/regression/util/DebugPoint.groovy]: done

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**